### DISCUSSION TITLE
Simply `currency` entity/table

### DISCUSSION THREAD

**Discussion by einar-hjortdal** [medusajs/medusa#3810] [Feature Requests]
There is no point in storing a whole lot of data regarding currencies. This data is not used by the backend and on the frontend `Intl.NumberFormat` exists (as well as other libraries such as Angular's currency pipe) and all we have to provide it is an ISO 4217 currency code.
Remove `symbol`, `symbol_native` and `name` from the `currency` entity.

**Comment by olivermrbl** [comment]
Are there any reasons for removing it besides the data not being used in the core? The currency information could be useful for plugins, integrations, custom functionality, etc. 

Closing this, as we will not include this in the roadmap.
